sources/packages/web/src/types/formio/custom-components/content/accepted-file-format-and-size.ts

```diff
@@ -0,0 +1,14 @@
+export const ACCEPTED_FILE_FORMAT_AND_SIZE = {
+  title: "Info - File format and size",
```

Comment:
Minor, we can call it "Uploader - File format and size" and later create some other ones like:
- "Uploader" (the uploader itself with the SIMS customization).
- "Uploader - Instructions"